Traceback (most recent call last):
  File "/scratch/jmille16/tylers_code/gpar2.py", line 130, in <module>
IOError: [Errno 5] Input/output error: '/scratch/jmille16/tylers_code/temp/saveplace_1.txt'
